WcaOnRails/config/locales/en.yml

```diff
@@ -343,6 +343,7 @@ en:
         event_restrictions: "I would like to restrict competitors from registering for specific combinations of events"
         event_restrictions_reason: "Reason and details for event restrictions"
         main_event_id: "Main event"
+        allow_registration_without_qualification: "Events with Qualification"
```

Comment:
The text here doesn't seem to fit the description. Where is this used?
(I'd expect something more along the lines of "Registering without automated checking of qualification" or "Registering without being qualified is allowed")